src/WebJobs.Script/BindingExtensions/ExtensionsManager.cs

```diff
@@ -278,7 +277,7 @@ private ProjectRootElement CreateDefaultProject(string path)
 
             root.AddItemGroup()
                 .AddItem(PackageReferenceElementName, ExtensionsPackageId)
-                .AddMetadata(PackageReferenceVersionElementName, "1.0.0-beta6", true);
+                .AddMetadata(PackageReferenceVersionElementName, "1.0.0", true);
```

Comment:
We should consider a force update to 1.0.0 for Metadata Generator.

Reply:
As we discussed, this is not on the update path, this simply applies the default references if a project doesn't exist.